be/src/exec/vectorized/hdfs_scanner_orc.cpp

```diff
@@ -20,7 +20,7 @@ class ORCHdfsFileStream : public orc::InputStream {
 
     uint64_t getLength() const override { return _length; }
 
-    uint64_t getNaturalReadSize() const override { return 8 * 1024 * 1024; }
+    uint64_t getNaturalReadSize() const override { return 2 * 1024 * 1024; }
```

Comment:
Could you elaborate a little bit why change it to `2`

Reply:
I don't have specific reason. The tradeoff is,  larger natural read size makes less IO counter, but makes larger memory footprint.

Reply:
I guess 2MB maybe is enough. The default value in apache/orc is 1MB.